### PR TITLE
Add netstandard16

### PR DESCRIPTION
**Pull request by joelverhagen**
Addresses https://github.com/NuGet/Home/issues/2536.

@emgarten @ericstj

### CODE REVIEW

**Comment by emgarten** [comment]
:shipit:

**Comment by ericstj** [comment]
LGTM.

File: src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs [hunk 1]
```diff
@@ -329,12 +329,15 @@ public IEnumerable<OneWayCompatibilityMappingEntry> CompatibilityMappings
...
-                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, new Version(1, 0, 0, 0)),
-                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, FrameworkConstants.MaxVersion)),
-                            new FrameworkRange(
-                                FrameworkConstants.CommonFrameworks.NetStandard10,
-                                FrameworkConstants.CommonFrameworks.NetStandard15))
+                        CreateStandardMapping(
+                            FrameworkConstants.CommonFrameworks.NetCoreApp10,
+                            FrameworkConstants.CommonFrameworks.NetStandard16),
+
+                        // NetCoreApp projects support NetStandard
```
Comment (weshaggard) [inline src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs:336]: nit: Comment is wrong for this one I assume that should say net463 supports netstandard.

Comment (joelverhagen) [inline src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs:336]: Yup! This change already went in so I will address this change separately. 
http://i.imgur.com/sVYTXTe.jpg

**Pull request merged**